tests/main.spec.ts

```
@@ -126,5 +126,31 @@ describe('monitor', () => {
         'scope.logs.success',
       );
     });
+
+    it('should write error log at trace level', () => {
+      const logger: BaseLogger = {
+        level: 'info',
+        info: jest.fn(),
+        debug: jest.fn(),
+        error: jest.fn(),
+        fatal: jest.fn(),
+        silent: jest.fn(),
+        trace: jest.fn(),
+        warn: jest.fn(),
+      };
+
+      setGlobalOptions({ logger, errorLogLevel: 'trace', parseError: (error) => error.message });
+
+      const scoped = createMonitor({ scope: 'scope' });
+
+      try {
+        scoped('logs', () => {
+          throw new Error('some error');
+        });
+        // eslint-disable-next-line no-empty
+      } catch {}
```

maybe just change this to expect to throw instead of the empty try catch